File: hotel_search/v2/get/by_offer/Python SDK/hotel_search.py
# Install the Python library from https://pypi.org/project/amadeus
from amadeus import Client, ResponseError

amadeus = Client(
    client_id='YOUR_AMADEUS_API_KEY',
    client_secret='YOUR_AMADEUS_API_SECRET'
)

try:
    # Confirm the availability of a specific offer
    offer_availability = amadeus.shopping.hotel_offer('F5B20A8E3C7EED38BDB931B587B3435232A8C804E3271D96F9B5826C326F61FA').get()
    print(offer_availability.data)
except ResponseError as error:
    raise error

File: hotel_search/v2/get/hotel_offers_offerId/Node SDK/hotel_search.js
var Amadeus = require("amadeus");
var amadeus = new Amadeus({
  clientId: 'YOUR_API_KEY',
  clientSecret: 'YOUR_API_SECRET'
});

// Confirm the availability of a specific offer
amadeus.shopping.hotelOffer('176383FB301E78D430F81A6CB6134EBF801DCC1AE14FC9DCCE84D17C6B519F5B').get()
  .then(function (response) {
    console.log(response);
  }).catch(function (response) {
    console.error(response);
  });